### PR TITLE
feat(relayer): adds additonal relayer droplet

### LINKED ISSUE #46: Relay Net machine
We need a new machine for relayer.
We'll switch from testnet naming to relaynet, so domain will be aries-relay-rpc-a, maybe different domains for bootstrap node URL and WS endpoint as well.

cc @isSerge

### PR DESCRIPTION
**Pull request by Genysys**
This PR adds an additional droplet for the relayer.

This PR closes #46

### CODE REVIEW

**Review by nazar-pc — DISMISSED** [review]

File: resources/aries-test-b/droplets.tf [hunk 1]
```diff
@@ -7,6 +7,21 @@ resource "digitalocean_droplet" "aries-test-b-nodes-farmer-relayer" {
     data.digitalocean_ssh_key.nazar-key.id,
     data.digitalocean_ssh_key.serge-key.id,
     data.digitalocean_ssh_key.leo-key.id,
+    # TODO: Add floating IP so that you can add this without changing the IP address
+    # data.digitalocean_ssh_key.sam-key.id,
+  ]
+}
+
+resource "digitalocean_droplet" "aries-relaynet-a" {
```
Comment (nazar-pc) [inline resources/aries-test-b/droplets.tf:13]: Why do we have `relaynet-a` in `test-b` file?

File: resources/aries-test-b/outputs.tf [hunk 2]
```diff
@@ -3,6 +3,11 @@ output "ip_address_test_droplet" {
   description = "ip_address_test_droplet: "
 }
 
+output "ip_address_test_droplet-2" {
```
Comment (nazar-pc) [inline resources/aries-test-b/outputs.tf]: This probably shouldn't be a test either.

File: resources/farmnet-a/droplets.tf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+resource "digitalocean_droplet" "aries-farmnet-a" {
```
Comment (nazar-pc) [inline resources/farmnet-a/droplets.tf:1]: Looks like this is a new instance we need for farmnet and not relaynet-related, right?

File: resources/farmnet-a/volumes.tf [hunk 4]
```diff
@@ -0,0 +1,12 @@
+resource "digitalocean_volume" "aries-farmnet-a" {
+  region                  = "nyc3"
+  name                    = "farmnet-a-volume"
+  size                    = 1800
+  initial_filesystem_type = "ext4"
+  description             = "Extra volume for relaynet a."
```
Comment (nazar-pc) [inline resources/farmnet-a/volumes.tf]: This is not a relaynet though, is it?

File: resources/aries-test-b/droplets.tf (hunk 1 above)
Comment (Genysys) [reply resources/aries-test-b/droplets.tf:13]: This was an error when it was deployed , and to change this now will bring down the current network , as the server would be redeployed

File: resources/farmnet-a/droplets.tf (hunk 3 above)
Comment (Genysys) [reply resources/farmnet-a/droplets.tf:1]: yes

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**